README.md

```diff
@@ -1,8 +1,64 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Adaobi Aniuchi . Jarvis Consulting
 
+I am a graduate student at the University of Windsor in the final days of my program. I am pursuing a professional master's degree in Applied Computing. Before that, I interned first as a Microsoft Leap Apprentice and subsequently as a graduate intern at a software development and consulting firm. During this time, I collaborated with the software team to release features for a custody management system for a local bank. The diverse range of applications in different industries possible by applying software development principles is one of my favorite things about the industry. I also like that it is an ever-evolving field, so there is always something new to learn. I am currently with Jarvis to expand and gain new software and data engineering skills by working on various software projects using different languages and frameworks in an agile environment.
```

Comment:
I am a graduate student at the University of Windsor in the final days of my program. I am pursuing a professional master's degree in Applied Computing. Before that, I interned first as a Microsoft Leap Apprentice and subsequently as a graduate intern at software development and consulting firm. During this time, I collaborated with the software team to release features for a custody management system for a local bank. The diverse range of applications in different industries possible by applying software development principles is one of my favourite things about the industry. I also like that it is an ever-evolving field, so there is always something new to learn. I am currently with Jarvis to expand and gain new software and data engineering skills by working on various software projects using different languages and frameworks in an agile environment.